src/TraceEvent/TraceEvent.csproj

```diff
@@ -55,6 +60,10 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent.SupportFiles" Version="$(MicrosoftDiagnosticsTracingTraceEventSupportFilesVersion)" />
     <PackageReference Include="Microsoft.Win32.Registry" Version="$(MicrosoftWin32RegistryVersion)" />
+    <PackageReference Include="PolySharp" Version="1.14.1">
```

Comment:
This package polyfills attributes for netstandard2.0

Reply:
Why not use latest version [1.15.0](https://www.nuget.org/packages/PolySharp/1.15.0)? Any breaking change that prevents usage?

Reply:
Probably works fine? I just did `nuget add package` and it did this

Reply:
wired that this command adds on older version and not the most recent one. Try the update and look what happens.